vnext/Microsoft.ReactNative/Fabric/Composition/CompositionViewComponentView.cpp

```diff
@@ -913,10 +913,13 @@ bool ComponentView::anyHitTestHelper(
   if (auto index = m_children.Size()) {
     do {
       index--;
-      targetTag = winrt::get_self<winrt::Microsoft::ReactNative::implementation::ComponentView>(m_children.GetAt(index))
-                      ->hitTest(ptContent, localPt);
-      if (targetTag != -1) {
-        return true;
+      auto child =
```

Comment:
What's `ComponentView::anyHitTestHelper` used for?  The function reads as "return true if any child view hit tests for the given point" and appears to avoid subRootViews like Modal with the rootComponentView check; will this disconnect the event chain for subRootViews at the subRoot instead of chaining through the Portal?

Reply:
anyHitTestHelper is basically an implementation detail of hitTest.  In any case, I undid this change, since now the modal's rootComponentView will be its own component view tree, rather than embedded in the same tree.  So we don't need to worry about all these extra checks when iterating the componentview tree any more.